dashboard/app/helpers/levels_helper.rb

```diff
@@ -74,6 +74,8 @@ def set_channel
   end
 
   def select_and_track_autoplay_video
+    return if @level.try(:never_autoplay_video) == "true"
```

Comment:
Thanks!  I'll add some tests like those, and see if I can get away with not serializing this property.

I do get the string "false" when this is false, or nil when it's never been set on the level in question.